sources/operator.1.16.0.psmdb-operator.json

```diff
@@ -0,0 +1,120 @@
+          "5.0.14-12": {
+            "image_path": "percona/percona-server-mongodb:5.0.14-12",
+            "image_hash": "38ced404ec798d78943c234ab4f39ee237f3013095a5cd581978cfdf0fbce2f9",
+            "status": "available",
+            "critical": false
+          },
+          "5.0.11-10": {
+            "image_path": "percona/percona-server-mongodb:5.0.11-10",
+            "image_hash": "da3713525d76a354435e1ab8fda12a06407e7eca8b8e72b9ac0163a34c8eb735",
+            "status": "available",
+            "critical": false
+          },
+          "5.0.7-6": {
+            "image_path": "percona/percona-server-mongodb:5.0.7-6",
+            "image_hash": "3f4849a17236c3849a513f46caa39fbc6da0414f98d27e080fbe0496fa9e86a2",
+            "status": "available",
+            "critical": false
+          },
+          "4.4.24-23": {
+            "image_path": "percona/percona-server-mongodb:4.4.24-23",
+            "image_hash": "87fa059b4b4460cbea9f79687e5cb158c9fc8fe75094fd6816657cef35701813",
+            "status": "recommended",
+            "critical": false
+          },
+          "4.4.19-19": {
+            "image_path": "percona/percona-server-mongodb:4.4.19-19",
+            "image_hash": "daeed5cc326cb6393d0304c247e96eb74f7844e30e2e38df84aa4368371bff81",
+            "status": "available",
+            "critical": false
+          },
+          "4.4.18-18": {
+            "image_path": "percona/percona-server-mongodb:4.4.18-18",
+            "image_hash": "fe34309f0491f13b4b6b35503f57391e57b1a87d88f6f842b1ca3606ddaf1e14",
+            "status": "available",
+            "critical": false
+          },
+          "4.4.16-16": {
+            "image_path": "percona/percona-server-mongodb:4.4.16-16",
+            "image_hash": "402b5e5b08ac73c74a47c72d002251a086f9ad28b0594fbae5c34757b294ce13",
+            "status": "available",
+            "critical": false
+          },
+          "4.4.13-13": {
+            "image_path": "percona/percona-server-mongodb:4.4.13-13",
+            "image_hash": "059c3c9a0360d6823905e39b52bdcaf76c3929c93408c537f139cee835c2bc0f",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.39.0": {
+            "image_path": "percona/pmm-client:2.39.0",
+            "image_hash": "4bae6c2fd94e0108a4204d6665ac72100a9cf5f786174ba040395759aebe47de",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "backup": {
+          "2.3.0": {
+            "image_path": "percona/percona-backup-mongodb:2.3.0",
+            "image_hash": "4b0a3a8dcd12619417aea16c7349cacacb41303b7131191ef65f4adff7ca1926",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "operator": {
+          "1.16.0": {
+            "image_path": "percona/percona-server-mongodb-operator:1.16.0",
+            "image_hash": "d8a5b33db1938d42769cb5a87d34a128332a2d0302eaa6d7c860e7c4667ea3b6",
```

Comment:
@tplavcic what should we put here since 1.16.0 is not released?

Reply:
I don't think we need PR for version service for mongodb 7.0 support, we can just update IMAGE_MONGOD_CHAIN in functions in the main repo.